dockerfiles/run/guice/memory/destination/conf/mailrepositorystore.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0"?>
+
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one
+  or more contributor license agreements.  See the NOTICE file
+  distributed with this work for additional information
+  regarding copyright ownership.  The ASF licenses this file
+  to you under the Apache License, Version 2.0 (the
+  "License"); you may not use this file except in compliance
+  with the License.  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing,
+  software distributed under the License is distributed on an
+  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  KIND, either express or implied.  See the License for the
+  specific language governing permissions and limitations
+  under the License.
+ -->
+
+<mailrepositorystore>
+    <defaultProtocol>file</defaultProtocol>
+    <mailrepositories>
+        <mailrepository class="org.apache.james.mailrepository.file.FileMailRepository">
+            <protocols>
+                <protocol>file</protocol>
+            </protocols>
+        </mailrepository>
+    </mailrepositories>
```

Comment:
Please add

```
    <mailrepositories>
        <mailrepository class="org.apache.james.mailrepository.memory.MemoryMailRepository">
            <protocols>
                <protocol>memory</protocol>
            </protocols>
        </mailrepository>
    </mailrepositories>
```

to gain access to Memory mail repositories

 - and drop me this aweful `FileMailRepository` + file protocol ^^